common/lib/xmodule/xmodule/static_content.py

```diff
@@ -186,6 +186,9 @@ def main():
     Generate
     Usage: static_content.py <output_root>
     """
+    from django.conf import settings
+    settings.configure()
```

Comment:
Who would use the settings?  The code that needs the settings (that caused the failure) are really far down the stack, we won't be passing them all the way down there.